app/src/main/java/com/github/swent/echo/compose/authentication/CreateProfile.kt

```diff
@@ -194,31 +228,45 @@ fun ProfileCreationUI(
                 modifier =
                     modifier.fillMaxSize().padding(16.dp).verticalScroll(rememberScrollState())
             ) {
-                // First name and last name fields
-                OutlinedTextField(
-                    value = firstName,
-                    onValueChange = { onFirstNameChange(it) },
-                    modifier = modifier.fillMaxWidth().testTag("FirstName"),
-                    label = {
-                        Text(text = stringResource(id = R.string.profile_creation_first_name))
-                    },
-                    singleLine = true,
-                    isError = firstName.isBlank()
-                )
-
-                Spacer(modifier = modifier.height(5.dp))
+                Row {
+                    Column {
+                        // First name and last name fields
+                        OutlinedTextField(
+                            value = firstName,
+                            onValueChange = { onFirstNameChange(it) },
+                            modifier =
+                                modifier
+                                    // .fillMaxWidth()
+                                    .testTag("FirstName"),
+                            label = {
+                                Text(
+                                    text = stringResource(id = R.string.profile_creation_first_name)
+                                )
+                            },
+                            singleLine = true,
+                            isError = firstName.isBlank()
+                        )
 
-                OutlinedTextField(
-                    value = lastName,
-                    onValueChange = { onLastNameChange(it) },
-                    modifier = modifier.fillMaxWidth().testTag("LastName"),
-                    label = {
-                        Text(text = stringResource(id = R.string.profile_creation_last_name))
-                    },
-                    singleLine = true,
-                    isError = lastName.isBlank()
-                )
+                        Spacer(modifier = modifier.height(5.dp))
```

Comment:
The height of the spacer can be set into a val at the beginning of the file, while it is use multiple times